build/pmm-server-docker/playbook-init.yml

```diff
@@ -113,19 +130,28 @@
     - name: PMM                        | Cleanup logs
       file: path={{ item }} state=absent
       with_items:
-        - /srv/logs/clickhouse.log
-        - /srv/logs/createdb.log
+        - /srv/logs/clickhouse-server.err.log
+        - /srv/logs/clickhouse-server.log
+        - /srv/logs/clickhouse-server.startup.log
         - /srv/logs/cron.log
         - /srv/logs/dashboard-upgrade.log
-        - /srv/logs/nginx.log
-        - /srv/logs/node_exporter.log
+        - /srv/logs/grafana.log
+        - /srv/logs/nginx.startup.log
+        - /srv/logs/pmm-agent.log
+        # - /srv/logs/pmm-configurator.log
```

Comment:
maybe we need to remove all commented lines

Reply:
pmm-configurator / pmm-manage logs are very useful while I work on updates, etc. I added them explicitly to remove later – when pmm-configuration is removed.